governance/votes/Executive vote - February 22, 2024.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Increasing wstETH Supply Cap on Spark Ethereum, Increasing the Spark DAI Effective Borrow APY, Housekeeping Actions, Triggering Spark Proxy Spell - February 22, 2024
```

Comment:
In past Executives, we've never included the SubDao (Spark) changes in the title, summary, or details. They always fall under the Spark Proxy Spell section

```suggestion
title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Triggering Spark Proxy Spell - February 22, 2024
```